File: app/src/main/java/com/example/alkewalletandroid/MainActivity7.java
package com.example.alkewalletandroid;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity7 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main7);

        ImageView volcerCuentaUsuario = findViewById(R.id.volver_cuenta);
        TextView volverCuentaUsuario2 = findViewById(R.id.volver_cuenta2);

        volcerCuentaUsuario.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity7.this, MainActivity5.class);
                startActivity(intent);
            }
        });

        volverCuentaUsuario2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity7.this, MainActivity5.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/alkewalletandroid/MainActivity7.kt
package com.example.alkewalletandroid

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity7 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main7)
        val volcerCuentaUsuario = findViewById<ImageView>(R.id.volver_cuenta)
        val volverCuentaUsuario2 = findViewById<TextView>(R.id.volver_cuenta2)
        volcerCuentaUsuario.setOnClickListener {
            val intent = Intent(this@MainActivity7, MainActivity5::class.java)
            startActivity(intent)
        }
        volverCuentaUsuario2.setOnClickListener {
            val intent = Intent(this@MainActivity7, MainActivity5::class.java)
            startActivity(intent)
        }
    }
}